content/docs/0.8.0/users-guide/default-disk-and-node-config.md

```diff
@@ -1,41 +1,78 @@
+3. Wait for Longhorn to sync the node tag automatically.
+
+> **Result:** If the node tag list was originally empty, Longhorn updates the node with the tag list, and you will see the tags for that node updated according to the annotation. If the node already had tags, you will see no change to the tag list.
+
+### Customizing Default Disks for New Nodes
+
+Longhorn uses the **Create Default Disk on Labeled Nodes** setting to enable default disk customization.
+
+- If the setting is enabled, Longhorn waits for the default disk customization to be set.
+- If the setting is disabled, Longhorn creates the default disk for new nodes that don't have existing disks. Then Longhorn relies on the value of the node label node.longhorn.io/create-default-disk to decide how to customize the default disks.
+
+If the value of the label is `config`, the annotation will be parsed and used as the default disk customization. If the value is the boolean value `true`, the data path setting will be used for the default disk. Other values will be treated as false and no default disk will be created.
+
+The default disk will not be automatically created on the new nodes unless the `create-default-disk` label is set.
 
-### Customize default disks for new nodes
+The configuration described in the annotation only takes effect when there are no existing disks or tags on the node.
+
+If the label or annotation fails validation, the whole annotation is ignored. 
+
+> **Prerequisite:** The Longhorn setting **Create Default Disk on Labeled Nodes** must be enabled.
+
+1. Add new nodes to the Kubernetes cluster, for example, by using Rancher or Terraform.
+2. Add the label to the node. Longhorn relies on the label to decide how to customize default disks:
 
-1. Enable the Longhorn setting `Create Default Disk on Labeled Nodes` to enable the feature. After the feature is enabled, the default disk will not be automatically created on the new nodes unless a certain label was set. 
-2. Add new nodes to the Kubernetes cluster, e.g. by using Rancher or Terraform, etc.
-3. Patch the label and annotation to each new node to define the default disks.
-    - Longhorn relies on the label to decide how to customize default disks. Since we choose using annotation here, the label should be:
     ```
     node.longhorn.io/create-default-disk: 'config'
     ```
-    - The annotation is used to specify the configuration of default disks. The format is `node.longhorn.io/default-disks-config: <disks configuration with JSON string format>`. For example:
+    
+3. Then add an annotation to the node. The annotation is used to specify the configuration of default disks. The format is:
+    
+    ```
+    node.longhorn.io/default-disks-config: <disks configuration with JSON string format>
+    ```
+    
+    For example, the following disk configuration can be specified in the annotation:
+    
     ```
     node.longhorn.io/default-disks-config: 
     '[{"path":"/mnt/disk1","allowScheduling":false},
```

Comment:
Let use this example instead:

```
node.longhorn.io/default-disks-config: 
'[{"path":"/mnt/disk1","allowScheduling":true},
  {"name":"fast-ssd-disk", "path":"/mnt/disk2","allowScheduling":false,"storageReserved":10485760,"tags":["ssd","fast"]}]'
```

We've added the ability to add name to disk recently. If user specify the same name for different disks, the configuration will be treated as invalid.

Reply:
For example, the following disk configuration can be specified in the annotation:
    
    ```
    node.longhorn.io/default-disks-config: 
    '[{"path":"/mnt/disk1","allowScheduling":true},
    {"name":"fast-ssd-disk", "path":"/mnt/disk2","allowScheduling":false,"storageReserved":10485760,"tags":["ssd","fast"]}]'
    ```

    > **Note:** If the same name is specified for different disks, the configuration will be treated as invalid.